omise/omise.php

```diff
@@ -40,4 +40,100 @@ public function __construct()
     }
+
+    public function getContent()
+    {
+        $fields_form[0]['form'] = array(
+            'legend' => array(
+                'title' => $this->l('Settings')
+            ),
+            'input'  => array(
+                array(
+                    'type'     => 'switch',
+                    'label'    => $this->l('Enable/Disable'),
+                    'name'     => 'module',
+                    'is_bool'  => true,
+                    'desc'     => $this->l('Enable Omise Payment Module.'),
+                    'values'   => array(
+                        array(
+                            'id'    => 'module_enabled',
+                            'value' => 1,
+                            'label' => 'Enabled'
+                        ),
+                        array(
+                            'id'    => 'module_disabled',
+                            'value' => 0,
+                            'label' => 'Disabled'
+                        )
+                    )
+                ),
+                array(
+                    'type'     => 'switch',
+                    'label'    => $this->l('Sandbox'),
+                    'name'     => 'sandbox',
+                    'is_bool'  => true,
+                    'desc'     => $this->l('Enabling sandbox means that all your transactions will be in TEST mode.'),
+                    'values'   => array(
+                        array(
+                            'id'    => 'sandbox_on',
+                            'value' => 1,
+                            'label' => 'Enabled'
+                        ),
+                        array(
+                            'id'    => 'sandbox_off',
+                            'value' => 0,
+                            'label' => 'Disabled'
+                        )
+                    )
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Public key for test'),
+                    'name'     => 'publicKeyForTest',
+                    'required' => false,
+                    'desc'     => 'The "Test" mode public key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Secret key for test'),
+                    'name'     => 'secretKeyForTest',
+                    'required' => false,
+                    'desc'     => 'The "Test" mode secret key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Public key for live'),
+                    'name'     => 'publicKeyForLive',
+                    'required' => false,
+                    'desc'     => 'The "Live" mode public key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
```

Comment:
@guzzilar Yes, that is right.

That page and feature has ALREDY in PrestaShop back-end. We do not need to develop anything for that permission feature.

Now the branches and pull requests are quite messy for me. I accepted that I create branches and pull requests not good.

I agree to change the type of secret key to be password type but if the changing in this point, the followed pull requests and files have affected and need to have more step in merge or rebase. That is I would like to prevent (for this situation).

If you want to do it in this point I let you to do it and solve the conflict for others pull requests.